### ISSUE TITLE
feat: add UI scaling for high DPI screens

### ISSUE THREAD

**Issue by CosmicRaptor** [ubuntu-flutter-community/musicpod#997]
I've got a 1080p 15" laptop and the text appears tiny. Having a way to enlarge the text and UI elements size would be very appreciated.

**Comment by CosmicRaptor** [comment]
No, that is way too big. I zoom firefox and element to 125% for example.

**Comment by Feichtmeier** [comment]
there should be this fractional scaling feature in ubuntu right under this 100 / 200 % steps
which distro do you use?

**Comment by CosmicRaptor** [comment]
Well I'm on Fedora 40 atm and its something added in GNOME 47. Will try on a F41 machine.
Regardless this could be great for other DE users who do not have fractional scaling.

**Comment by Feichtmeier** [comment]
sure, okay I'll have a look 
we could add a text slider and then reload the current theme with an override of the text theme with new values scaled or something
